library/core/src/mem/maybe_uninit.rs

```diff
@@ -823,10 +816,7 @@ impl<T> MaybeUninit<T> {
     ///     foo.assume_init()
     /// };
     /// ```
-    // FIXME(#76092): We currently rely on the above being incorrect, i.e., we have references
-    // to uninitialized data (e.g., in `libcore/fmt/float.rs`).  We should make
```

Comment:
Why did you remove this FIXME? I don't think it has been resolved, has it?

Reply:
https://github.com/rust-lang/rust/pull/76241 fixed this for `fmt/float.rs`, but before removing the FIXME we need an audit of all the uses of these functions in the standard library. Note the "e.g.", this was not an exhaustive list.

Reply:
Oh I thought it was fixed as the issue was closed. Then should we re-open it or have a new issue? And I still don't understand this very well, so if you could explain it to me, that would be great.

Reply:
Okay I greped the code and this is the only piece I found, other uses have debug assertions or safety comments:
https://github.com/rust-lang/rust/blob/14831568d506b5ee7be3e7d5a2f029ce9048b609/library/std/src/io/copy.rs#L128-L156

Can this be a blocker to be stabilized?

Reply:
I agree the other uses are fine, and this code will hopefully disappear when https://github.com/rust-lang/rfcs/blob/master/text/2930-read-buf.md gets implemented.

So, fine to remove the FIXME. :)